docs/core/extensions/configuration-providers.md

```diff
@@ -25,6 +25,9 @@ Configuration in .NET is possible with configuration providers. There are severa
 - [XML configuration provider](#xml-configuration-provider)
 - [INI configuration provider](#ini-configuration-provider)
 
+> [!IMPORTANT]
+> Keys are case-insensitive, so be sure that your keys are unique. All of the file configuration providers throw the <xref:System.FormatException> when duplicate keys are found.
```

Comment:
```suggestion
Keys are case-insensitive. All of the file configuration providers throw the <xref:System.FormatException> when duplicate keys are found in a single provider. For example, if your _appsettings.json_ file contains duplicate keys it will error:

```json
{
    "settings": "value-1",
    "SETTINGS": "value-2"
}
```
```